src/modules/roc_pipeline/receiver_session.cpp

```diff
@@ -35,7 +36,8 @@ ReceiverSession::ReceiverSession(const SessionConfig& config,
 
     if (config.resampling) {
         resampler_updater_.reset(new (allocator_) audio::ResamplerUpdater(
-                                     config.fe_update_interval, config.latency),
+                                     config.fe_update_interval, config.latency,
+                                     (float)format->sample_rate / out_sample_rate),
```

Comment:
We should panic if out_sample_rate is zero. A panic is better than a segfault.

Reply:
Fixed

Reply:
Fixed

Reply:
Fixed